internal/service/environment.go

```diff
@@ -23,3 +40,36 @@ func (e *Environment) ListEnvironments(ctx *context.Context, request *environmen
 
 	return response, nil
 }
+
+// CreateEnvironment creates environment
+func (e *Environment) CreateEnvironment(ctx *context.Context, request *environment.CreateEnvironmentRequest) error {
+	conn, requestCtx, err := grpcClient(ctx)
+	if err != nil {
+		return err
+	}
+	client := environment.NewEnvironmentServiceClient(conn)
+	stream, err := client.CreateEnvironment(*requestCtx, request)
+	if err != nil {
+		return err
+	}
+
+	log.Info("Creating environment...")
+	spinner := spinner.New(spinner.CharSets[SpinnerType], SpinnerDelay)
+	spinner.Color(SpinnerColor, SpinnerStyle)
+
+	for {
```

Comment:
If multiple accounts are passed then we have a different status for each of the account. So ideally there should be a multi spinner, one for each account

Reply:
For multiple accounts, env creation is considered successful only when it is created in all the mentioned account. So no need for account specific status